### ISSUE TITLE
Use Teams to control access level?

### ISSUE THREAD

**Issue by arshad** [backdrop-ops/contrib#36]
Would using Team be a good idea for controlling access to repos? 

See https://help.github.com/articles/adding-or-inviting-members-to-a-team-in-an-organization/

Team allow access at a repo level instead of organization level. So if you have a team named **Captcha Team** and add users to it. They can only read and write to that repo.

We might need to setup a team per repo though.

**Comment by quicksketch** [comment]
We had considered that sort of arrangement initially, but it looked like it was going to add a lot of maintenance overhead without a lot of value. It indeed would mean that every project would need its own team.

The reason we didn't do that is because in order to create a new project, the user must have "Admin" privileges (that's what everyone has now). But a user can't administer the teams unless they are an "Owner" in the overall organization. That means we would have needed pretty much every contrib author to be an owner so they could manage their own teams, and you'd end up with hundreds of different teams.

So basically, the way it's set up now, everyone is an "Admin". If people needed to manage their own teams, everyone would need to be an "Owner". GitHub's system just isn't set up for large-scale collaboration like this, so we have to manage with "unofficial" mechanisms like the contributor agreement, and listing the maintainers in the README.md as the "official" list of committers.

**Comment by arshad** [comment]
Thanks for the info. I think this can be closed for now.
